HOOKS.md

```diff
@@ -203,41 +221,28 @@ This hook configuration will execute hook on each change in labels of pods label
 
 > Note: Shell-operator requires a ServiceAccount with the appropriate [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions. See examples with RBAC: [monitor-pods](examples/101-monitor-pods) and [monitor-namespaces](examples/102-monitor-namespaces).
 
-## Main cycle: events tracking and hooks execution
-
-Events may happen at any time, so the working queue for hooks is provided in the Shell-operator:
-
-- Adding events to the queue:
-  - `onStartup` hooks are added to the queue during initialization,
-  - `onKubernetesEvent` hooks are added to the queue when events occur in Kubernetes,
-  - `schedule` hooks are added according to the schedule.
-
-- Queue handler executes hooks strictly sequentially. If hook fails with an error (non-zero exit code), Shell-operator restarts it (every 5 seconds) until success. In case of an erroneous execution of some hook, when other events occur, the queue will be filled with new tasks, but their execution will be blocked.
-  - You can change this behavior for a specific hook by adding `allowFailure: true` to the binding configuration (not available for `onStartup` hooks).
-
-- Several metrics are available for monitoring the activity of a queue: queue size, number of execution errors for specific hooks, etc. See [METRICS](METRICS.md) for more details.
+> Note: consider that "Added" event is not always equal to "Object created" if labelSelector, fieldSelector or namespace.labelSelector is specified. Object or namespace is updated and labels are matched the selector, but API returns "Added" event in this case. The same with "Deleted" event: "Deleted" is not always equal to "Object removed", the object can just become out of scope.
 
 ## Event triggered execution
 
-When an event associated with a hook is triggered, Shell-operator executes the hook without arguments and sets the following environment variables:
+When an event associated with a hook is triggered, Shell-operator executes the hook without arguments and sets the following environment variable:
 
 - `BINDING_CONTEXT_PATH` — a path to a temporary file containing data about an event that was triggered (binding context);
 
-- `WORKING_DIR` — a path to a hooks directory (see [an example](examples/003-common-library) — using libraries via `WORKING_DIR`).
-
 ### Binding context
 
-Binding context is the information about the event that led to the hook execution.
+When an event associated with a hook is triggered, Shell-operator  executes this hook without arguments. The information about the event that led to the hook execution is called a binding context and is written to a temporary file. `BINDING_CONTEXT_PATH` environment variable contains the path to this file with JSON-array of structures with the following fields:
```

Comment:
>is called a binding context

Is called the **binding context**.

We should emphasize on an important term that used throughout the shell-operator.

Reply:
>  important term 
It is in the header...

Reply:
This changes nothing.